src/args.rs

```diff
@@ -39,6 +39,10 @@ pub struct ProxyboiConfig {
     #[structopt(parse(try_from_str = from_url))]
     pub upstream: Url,
 
+    /// Connection timeout against upstream (including DNS name resolution)
```

Comment:
You should also declare that this is seconds as opposed to milliseconds, for instance.